<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thorlabs Qt GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/favicon.png</normaloff>:/images/favicon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSlider" name="sliderX">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="ncm_qapt_gui_graphicsview" name="graphicsView"/>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="sliderY">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDial" name="dialZ">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QComboBox" name="controllerX"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="controllerY"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X axis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="controllerZ"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y axis</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z axis</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="spinX">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="spinY">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="spinZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="homeX">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="homeY">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="homeZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ncm_qapt_gui_graphicsview</class>
   <extends>QGraphicsView</extends>
   <header>ncm_qapt_gui_graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ncm_qmarkup.qrc"/>
 </resources>
 <connections/>
</ui>
